odoo/addons/base/models/ir_asset.py

```diff
@@ -348,7 +348,7 @@ def is_safe_path(path):
                     return False
                 if path.rpartition('.')[2] in TEMPLATE_EXTENSIONS:
                     # Forbid xml to leak
-                    return ("%s/static/" % addon) in path
+                    return (os.path.normpath("%s/static" % addon) + os.path.sep) in path
```

Comment:
This should have a comment explaining that `normpath` will strip the trailing `/`, which is why it has to be added afterwards.